change/@fluentui-react-card-97d7abde-06d9-46cc-a71a-147630a0431e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28968&commitId=eb3032532b877f21cbac0d091b388ed6a45bbbae&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.card templates.normal.chromium.png|2774| Changed|
|Card Converged.appearance.normal.chromium.png|35403| Changed|
|Card Converged.CardHeader.normal.chromium.png|11378| Changed|
|Card Converged.appearance - High Contrast.normal.chromium.png|40980| Changed|
|Card Converged.appearance - Dark Mode.normal.chromium.png|24215| Changed|
|Card Converged.appearance - RTL.normal.chromium.png|35411| Changed|
|Card Converged.size.normal.chromium.png|23648| Changed|
|Card Converged.orientation.chromium.png|6216| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|3387| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|3292| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|3402| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|4611| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|5269| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|3387| Changed|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|4160| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|3121| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|5269| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|4066| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|3387| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|3402| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|3187| Changed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|4712| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|3198| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|3198| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|3449| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|5269| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|3293| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|3472| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|3437| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|4444| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|4513| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|5269| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|3164| Changed|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|4712| Changed|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|4713| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|4161| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|3387| Changed|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|4713| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|4712| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|3240| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|3387| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|3566| Changed|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|4513| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|3387| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 68 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|4712| Changed|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|4356| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|3402| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|4156| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|3297| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|3198| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|5269| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|4352| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|3292| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|4507| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.click.chromium.png|3566| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|3121| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|4061| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.normal.chromium.png|5269| Changed|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|4513| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.hover.chromium.png|3187| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|4439| Changed|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|4160| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|4707| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.hover.chromium.png|3387| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|3387| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.click.chromium.png|3387| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|4263| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|4611| Changed|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|4513| Changed|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|4356| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|4606| Changed|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|4160| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|4066| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.selected.chromium.png|3198| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.selected.chromium.png|3396| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.click.chromium.png|3449| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.selected.chromium.png|3472| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.normal.chromium.png|3437| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|4268| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|3387| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|4444| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.click.chromium.png|3198| Changed|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|4712| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.selected.chromium.png|3198| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|4708| Changed|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|4713| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.click.chromium.png|3240| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.normal.chromium.png|3293| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.normal.chromium.png|3402| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.hover.chromium.png|3198| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.normal.chromium.png|5269| Changed|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|4713| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|3198| Changed|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|4712| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|4707| Changed|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|4713| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|4707| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|5269| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.hover.chromium.png|4508| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|4712| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.normal.chromium.png|4707| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|4161| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|4513| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|3387| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.hover.chromium.png|3472| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.click.chromium.png|4157| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|4357| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.hover.chromium.png|3164| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|4707| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|3387| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.selected.chromium.png|3331| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.selected.chromium.png|4353| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Resolved as this is intended to happen. The card header now has less space between the elements which result in the whole rest of the content shifting to the top.